src/huggingface_hub/inference/_providers/hf_inference.py

```diff
@@ -125,18 +126,27 @@ def _prepare_url(self, api_key: str, mapped_model: str) -> str:
 
 
 def _build_chat_completion_url(model_url: str) -> str:
-    # Strip trailing /
-    model_url = model_url.rstrip("/")
+    parsed = urlparse(model_url)
+    path = parsed.path
```

Comment:
```suggestion
    path = parsed.path.rstrip("/")
```

(so that it's done?)